### ISSUE TITLE
SIRIUS and NAP fails with IIN mgf

### ISSUE THREAD

**Issue by irinakoester** [CCMS-UCSD/GNPS_Workflows#388]
My SIRIUS run failed (with or without ZODIAC) and so did NAP. Maybe it's a problem because I used the mgf file after IIN?
Sirius: ID=52828118369c4f1cbad439251035bbef
Sirius+ Zodiac: ID=1b91a880daaa4db2afc895976e0c0e3f
NAP: ID=87c25c67717746a5a37a9d9d00c12a33

**Comment by mwang87** [comment]
I would recommend not doing IIN as it is known to sometimes do weird renumberings.

**Comment by robinschmid** [comment]
I have just fixed the renumbering issue. It was accidentally introduced with Kai's spectral merging function (that he ported for my IIN version of MZmine).

You can download from my github. Nothing was changed for IIN.
https://github.com/robinschmid/mzmine2/releases/tag/corr_17.7_library_stats

**Comment by mwang87** [comment]
Do you know when the fix will go out in the mainline MZMine version?

**Comment by robinschmid** [comment]
The main version does not need a fix...
First we are writing MZmine 3 with a more modular data model, then I need to implement the grouping and IIN for the new data model. 
This will take a bit.

**Comment by mwang87** [comment]
Oh, is IIN only in your particular branch of MZMine2 for now?

**Comment by robinschmid** [comment]
Yeah the problem is that IIN introduces a completely new data structure - and my plan was to integrate this into the main branch last December in Boston with Tomas - however we decided to first work on MZmine 3 and reprogram the feature data structure from scratch. 
MZmine 3 is going to add plenty of nice features. We are still working on it.
